app/lib/services.rb

```diff
@@ -11,6 +11,10 @@ def self.publishing_api
 
   def self.rummager
     @rummager ||=
-      GdsApi::Rummager.new(Plek.current.find('search'), disable_cache: true, api_version: 'V2')
+      GdsApi::Rummager.new(Plek.current.find('search'),
+                           disable_cache: true,
```

Comment:
I think you can drop this option as it no longer has any effect since: https://github.com/alphagov/gds-api-adapters/pull/848

Reply:
👍 Fixed in b48fe65